docs/user/gcp/install_upi.md

```diff
@@ -98,6 +105,19 @@ del data["spec"]["privateZone"];
 open(path, "w").write(yaml.dump(data, default_flow_style=False))'
 ```
 
+### Update the cloud-provider manifest (optional)
+If you are installing into a Shared VPC, update the cloud provider configuration so it understands the network and subnetworks are in a different project (host project).
+The service account used to install the cluster, and the master/worker service accounts created later in this document must all have `roles/compute.networkUser` access to each of the host project subnets used by the cluster.
+This can be achieved using `compute networks subnets set-iam-policy`.
+
+```sh
+export HOST_PROJECT="example-shared-vpc"
+export HOST_PROJECT_NETWORK_NAME="example-network"
+export HOST_PROJECT_COMPUTE_SUBNET_NAME="example-worker-subnet"
+
+sed -i "s/    subnetwork-name.*/    network-project-id = ${HOST_PROJECT}\\n    network-name    = ${HOST_PROJECT_NETWORK_NAME}\\n    subnetwork-name = ${HOST_PROJECT_COMPUTE_SUBNET_NAME}/" manifests/cloud-provider-config.yaml
+```
```

Comment:
this is fine, but i think i would rather document what the users need to do in terms of what the end result should look like.
a way to achieve this is good in addition to above.

Reply:
Updated with 998a518a17e54a792bd52cf31b217775eb20f5a3